Cargo.toml

```diff
@@ -4,6 +4,8 @@
 
 [workspace]
 
+resolver = "2"
```

Comment:
Maybe add a comment for resolver to explain why it's needed

Reply:
Fixed.